.github/workflows/python-release-minor.yml

```diff
@@ -76,3 +76,14 @@ jobs:
           PYPI_USERNAME: ${{ secrets.PYPI_USER }}
           PYPI_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
         run: poetry publish -u $PYPI_USERNAME -p $PYPI_PASSWORD -v -n
+
+      - name: Bump repo version
+        run: |
+          git clean -fxd # Clear all the changes
+          poetry version ${{ env.short_version }}.dev0
+
```

Comment:
I see, so we bump the version after we publish, which would mean that `.dev0` is the current version we're working on. Does poetry automatically remove `.dev0` from version name when publishing?